### DISCUSSION TITLE
Whats next for Hoot Draft?

### DISCUSSION THREAD

**Discussion by mattheworres** [mattheworres/phpdraft#76] [General]
Hello all!

I hope this message finds everyone safe and secure - 2020 has been, something - but I wish everyone safety and happiness as we approach the conclusion of the year.

Before I get to where the project goes, I need to explain why the project went nowhere this year. 2020, of all years, when at all possible folks stay home, I barely committed to the project save to fix a few items. Why? The project is still alive and well, I just had a difficulty in finding the time and motivation to hunker down and get it done. In part: in December of 2019, I began a position with Verizon Media to work on the Yahoo! Fantasy Sports web app!

First - Hoot Draft is not dead! When coming aboard at Y! I made sure that my projects could remain my projects - the main thing was that I was starting a new position and for most of the last year I've been adjusting to learning multiple codebases for a product that I was now working on! This year, though unprecedented, personally and professionally has been a year of wonderful growth and happiness and I look forward to continuing in my role at Y! Sports to continue working on and maintaining the great products and services we provide the fantasy community.

As for Hoot Draft? I have a fairly big rewrite planned in a few big stages to allow both Keepers and salary cap-style (previously "auction") drafts. The work seems fun, I just will need to hunker down to actually get it implemented.

In the meantime, I have unfinished code sitting on my laptop that I still need to complete (quality of life stuff around the main table that has been bugging me for literal years) before i can move to the exciting new features.

If you have any feature requests, or ideas for new features, please post! Thanks everyone, have a Happy Holidays and a Happy New Year!

**Comment by vvp1187** [comment]
I'm sorry if this isn't the best place to post this. I'm new to github. I had tried to sign up at hootdraft.com, but the human check seems to not be working, and the register button was not able to be clicked. Was hoping to use the website to set up a draft. Thank you!

**Comment by mattheworres** [comment]
Ack! You have indeed found a bug, I must have broken the Registration page
with the latest upgrade. I'll try to get that fixed up for you tonight.
Sorry about that!

**Reply by vvp1187** [reply]
Awesome! I'll try again later. Thank you so much!
